index.js

```diff
@@ -114,31 +112,32 @@ module.exports = function (spec) {
     if (results.tests.length === 0) {
       return pad(format.red(symbols.cross + ' No tests found'));
     }
-
-    return _.filter([
+ 
+    const temp = [
       pad('total:     ' + results.asserts.length),
       pad(format.green('passing:   ' + results.pass.length)),
       results.fail.length > 0 ? pad(format.red('failing:   ' + results.fail.length)) : undefined,
       pad('duration:  ' + prettyMs(new Date().getTime() - startTime))
-    ], _.identity).join('\n');
+    ];
+    return temp.join('\n');
```

Comment:
Missing a `.filter` to account for line 119 which may returned `undefined`.

Reply:
see #59